src/AppBundle/Controller/BlogController.php

```diff
@@ -34,12 +34,20 @@
 class BlogController extends Controller
 {
     /**
-     * @Route("/", name="blog_index")
+     * @Route("/", name="blog_index", defaults={"page" = 1})
+     * @Route("/page/{page}", name="blog_index_paginated", defaults={"page" = 1}, requirements={"page" : "\d+"})
```

Comment:
After some refactoring with separate route definitions you have a point now.
We should avoid defaults for page in second route annotation.